core/iwasm/common/wasm_memory.c

```diff
@@ -233,6 +248,85 @@ wasm_runtime_create_shared_heap(SharedHeapInitArgs *init_args)
     return NULL;
 }
 
+WASMSharedHeap *
+wasm_runtime_chain_shared_heaps(WASMSharedHeap *head, WASMSharedHeap *body)
```

Comment:
I'm not sure if it's just me, but the names head and body are confusing. Typically, head refers to the fixed part, and body is the part that you want to append to head. In our case, it seems to be the opposite.

Reply:
I will add more content to the document in the following PR